eventkitd/main.go

```diff
@@ -63,35 +59,16 @@ func handleParsedPacket(packet *pb.Packet, source *net.UDPAddr, received time.Ti
 	return nil
 }
 
-func handlePacket(packet UDPPacket) error {
-	defer packet.Return()
```

Comment:
this defer is important

Reply:
it's preserved here: https://github.com/jtolio/eventkit/pull/5/files#diff-e86e601ae5292817399a481c2c0ed49aeb67b393ac0cddc00d4aa31c491b40b1R48-R53